payments-model/src/test/java/com/stripe/android/repository/ConsumersApiServiceImplTest.kt

```diff
@@ -93,6 +131,14 @@ class ConsumersApiServiceImplTest {
         )
     }
 
+    @Test
+    fun testStartConsumerVerificationUrl() {
```

Comment:
You can make this pass by adding a `ApiRequest.apiTestHost = null` to the start of this function.

Reply:
https://github.com/stripe/stripe-android/pull/6132/commits/1338675609e94b790f9b6edf99a6ebde82c11a14